client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -64,8 +74,17 @@ export const receiveError = ( { dispatch }, { siteId, postId }, next, rawError )
  * @param {Array} revisions raw data from post revisions API
  */
 export const receiveSuccess = ( { dispatch }, { siteId, postId }, next, revisions ) => {
+	const normalizedRevisions = map( revisions, normalizeRevision );
+
+	forEach( normalizedRevisions, ( revision, index ) => {
+		revision.changes = index === normalizedRevisions.length - 1
+			? { added: 0, removed: 0 }
+			: countDiffWords( diffWords( normalizedRevisions[ index + 1 ].content, revision.content ) );
+	} );
+
 	dispatch( receivePostRevisionsSuccess( siteId, postId ) );
-	dispatch( receivePostRevisions( siteId, postId, map( revisions, normalizeRevision ) ) );
+	dispatch( receivePostRevisions( siteId, postId, normalizedRevisions ) );
+	dispatch( requestUsers( siteId, uniq( map( normalizedRevisions, 'author' ) ) ) );
```

Comment:
Do we really want to query the users as soon as we receive the revisions? My first thought is that we should probably use a query component for this when needed. Is there a particular reason for this to be fetched here?